cap/config.py

```diff
@@ -91,9 +91,18 @@ def _(x):
 #: Email address for support.
 SUPPORT_EMAIL = "analysis-preservation-support@cern.ch"
 #: Disable email sending by default.
-MAIL_DEBUG = False
+MAIL_DEBUG = True
```

Comment:
No need to specific here.. They can be overwritten from DEFAULTS or with envvars `INVENIO_***` style